packages/js/plugins/uri-resolvers/ipfs-resolver/src/utils/IpfsClient.ts

```diff
@@ -0,0 +1,19 @@
+import CID from "cids";
+
+export interface IpfsClient {
+  add(data: Uint8Array, options?: unknown): Promise<IpfsFileInfo[]>;
```

Comment:
Probably this is non sense but should the interface of `IpfsClient` defined in the `ipfs-resolver` have the `add` method? I guess it should just be aware of getting content

Reply:
This interface should be only for tests and your comment made me check that and find a larger issue, so very nice catch :D